tests/store/test_boundary_conditions.py

```diff
@@ -1,12 +1,18 @@
 import numpy as np
-from helpers import get_bc_datapath
+import pytest
+from helpers import get_bc_datapath,clear_test_stores
 from openghg.retrieve import search
 from openghg.standardise import standardise_bc, standardise_from_binary_data
 from openghg.store import BoundaryConditions
 from openghg.util import hash_bytes
 from xarray import open_dataset
 
 
+@pytest.fixture
```

Comment:
Do we need to clear all the test stores before each test?

Reply:
I removed the clear test store fixture from some of the tests and they still run fine. We should try and limit the number of times we call that function. If we really need to clear a store there is the `clear_test_store` function that just clears a specific store.